src/client/client.ts

```diff
@@ -495,6 +630,22 @@ export class DeskproClient implements IDeskproClient {
       this.getStaticOAuth2Token = (key: string) => parent._getStaticOAuth2Token(key);
     }
 
+    if (parent._startOAuth2Flow) {
```

Comment:
Typings say this always exist so either this check isn't required or typings need updating to say optional.